src/BoE-Info.plist

```diff
@@ -151,21 +151,25 @@
 		</dict>
 	</array>
 	<key>CFBundleExecutable</key>
-	<string>Blades of Exile</string>
+	<string>$(EXECUTABLE_NAME)</string>
 	<key>CFBundleIconFile</key>
 	<string>BoE</string>
 	<key>CFBundleIdentifier</key>
-	<string>com.spidweb.bladesofexile</string>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
```

Comment:
I might've purposely hard-coded this so that all three programs use the same domain...